pkgs/applications/virtualization/docker/default.nix

```diff
@@ -60,17 +60,12 @@ rec {
       NIX_CFLAGS_COMPILE = "-DMINIMAL=ON";
     });
 
-    moby = buildGoPackage ((optionalAttrs (stdenv.isLinux)) rec {
+    moby = buildGoPackage ((optionalAttrs (true)) rec {
```

Comment:
Looks like you've accidentally modified this line (for testing?)

Reply:
Ah yeah 🤦‍♂️ . Fixed.